apps/userdashboard/assets/js/a4_candy_userdashboard/UserReport.jsx

```diff
@@ -23,7 +23,7 @@ export const UserReport = (props) => {
     return (
       <p>
         {splitted[0]}
-        <b>{splitted[1]}</b>
+        <b>{userName}</b>
```

Comment:
I dunno why this got altered by the py linter. @goapunk anyhow it is correct.

Reply:
ups, nice catch!